bin/postgres-ha/bootstrap-postgres-ha.sh

```diff
@@ -84,6 +84,14 @@ initialization_monitor() {
                 sleep 1
             done
 
+            if [[ "${manual_start}" == "true" ]]
+            then
+                echo_info "Executing Patroni restart to restart database and update configuration"
+                curl -X POST --silent "127.0.0.1:${PGHA_PATRONI_PORT}/restart"
```

Comment:
@andrewlecuyer Why are we not calling `patronictl`?

Reply:
Since we have not moved to `patronictl` elsewhere in the container, for now I decided to keep it consistent with the other calls to Patroni.

Additionally, this is essentially the same exact code as was previously used to perform a restart at this point, so considering our timeline for 4.4 I decided to stick with something that had been previously tested and utilized.